frontend/src/bundles/common/components/form-group/form-group.tsx

```diff
@@ -0,0 +1,31 @@
+import { type ReactNode } from 'react';
+import { type FieldError } from 'react-hook-form';
+
+import styles from './styles.module.scss';
+
+type Properties = {
+    width?: string;
+    error: FieldError | undefined;
+    label: string;
+    children: ReactNode;
+};
+
+const FormGroup = ({
+    error,
+    label,
+    children,
+    width = 'auto',
+}: Properties): JSX.Element => {
+    const errorMessage = error?.message;
+    return (
```

Comment:
It is a common practice to add space before return statement to increase redability
```suggestion
    const errorMessage = error?.message;
    
    return (
```